### DISCUSSION TITLE
is it working for anycubic i3 mega s?

### DISCUSSION THREAD

**Discussion by TeleportYT** [Sakati84/3DPrintColorizer#49] [Q&A]
hi, i saw your project and its just sooooo impressive and i am wondering is it working for anycubic i3 mega s?

**Comment by Sakati84** [comment]
If you can mount a BLTouch to your print head it should work. You can mount the holder to the BLTouch mount and fix the pen rack with zip ties. Just have a look at the Wiki.

**Reply by TeleportYT** [reply]
Okay , thanks :)
